src/absil/illex.fsl

```diff
@@ -119,21 +115,21 @@ rule token = parse
   (* The problem is telling an integer-followed-by-ellipses from a floating-point-nubmer-followed-by-dots *)
 
   | ((['0'-'9']) | (['0'-'9']['0'-'9']['0'-'9']+)) "..."
-      {  let b = lexeme lexbuf in 
-         VAL_INT32_ELIPSES(int32(String.sub b  0 (String.length b - 3))) }
+      {  let b = lexbuf.LexemeView in 
+         VAL_INT32_ELIPSES(int32(b.Slice(0, (b.Length - 3)).ToString())) }
```

Comment:
This change is new

Reply:
Worth noting that this doesn't improve performance since the string allocation still happens. But I suppose it's not harmful to have it.

Reply:
Well I suppose it might, if the LexemeView doesn't allocate.

Reply:
yes the LexemeView is a span so this should save one of two allocations.